v2/tools/generator/internal/codegen/pipeline/testdata/TestAddOperatorSpec_AddsSpecWithConfiguredSecrets/person-v20200101.golden

```diff
@@ -0,0 +1,58 @@
+// Code generated by azure-service-operator-codegen. DO NOT EDIT.
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+package v20200101
+
+import (
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+type Person struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              Person_Spec   `json:"spec,omitempty"`
+	Status            Person_Status `json:"status,omitempty"`
+}
+
+// +kubebuilder:object:root=true
+type PersonList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []Person `json:"items"`
+}
+
+type Person_Spec struct {
+	//FamilyName: Shared name of the family
+	FamilyName string `json:"familyName"`
+
+	//FullName: As would be used to address mail
+	FullName string `json:"fullName"`
+
+	//KnownAs: How the person is generally known
+	KnownAs string `json:"knownAs"`
+
+	//OperatorSpec: The operator spec
+	OperatorSpec *PersonOperatorSpec `json:"operatorSpec,omitempty"`
+}
+
+type Person_Status struct {
+	Status string `json:"status"`
+}
+
+type PersonOperatorSpec struct {
```

Comment:
Give this type a description so that someone reading the generated code knows what it's for.